example/lib/main.dart

```diff
@@ -51,20 +55,23 @@ void main() {
 ///Initialize function is always called at the start of your application or within
 ///the scope of use of LexActivator
 void initializeLexActivator() {
-  LexActivator.SetProductData(productData: "PASTE_CONTENT_OF_PRODUCT.DAT_FILE");
+  LexActivator.SetProductData(productData: "SET_PRODUCT_DATA");
```

Comment:
@mumin-khan PASTE_CONTENT_OF_PRODUCT.DAT_FILE is what we have used across other examples, so we should be doing the same here too.